contents/handbook/people/team-structure/pipeline/objectives.mdx

```diff
@@ -0,0 +1,69 @@
+- Objective: Develop Posthog CDP into a solution capable of replacing other CDPs for data-warehousing categories. Increase % of customers using “data out” apps from X to Y
+    - Key result:
+        - Make the most important data in and out apps work reliably
+            - (Potentially) Consolidate into 1 repo
+            - Opinionated way of writing tests (maybe end-to-end), structuring the apps
+            - Separate out filters
+        - Team to fill in
+    - Ideas
+        - CDP style UI
```

Comment:
I'd argue strongly against any extensive UI/UX work from our side for Q4 specifically.

We have a good amount of work to do on the scaffolding still (job queues, plugin testing, horizontal scalability) that this would stretch us out very wide. 

Note that @pjhul seems quite interested in working on the more user-facing side of apps, both during Q4 and longer term (from my understanding). This is something we can collaborate on.

Reply:
Definitely agree with this. I think it's better for ingestion team to focus on the pipeline itself rather than building UI or maintaining apps. Recently, I've been focusing more on this and like Yakko mentioned I'd be very interested in continuing to look at this more deeply. Not sure exactly how this would fit in since there isn't really an 'apps' team, but if this does stay as part of the Pipeline team objectives I'd be happy to collaborate on this.